src/quantum/plugin/QuantumOptions.ts

```diff
@@ -222,7 +222,10 @@ export class QuantumOptions {
         return this.optsTarget === "electron";
     }
     public isTargetUniveral() {
-        return this.optsTarget === "universal";
+        return this.optsTarget === "universal" || this.optsTarget === "npm";

```

Comment:
That's not correct, universal - means it will be working on browser too, where there is no `require`